diff --git a/Makefile b/Makefile
index 67913fc96..77c76b615 100644
--- a/Makefile
+++ b/Makefile
@@ -538,9 +538,6 @@ install:
 	$(MKDIR) "$(INSTALL_LIBDIR)"
 	$(MKDIR) "$(INSTALL_STUBLIBDIR)"
 	$(MKDIR) "$(INSTALL_COMPLIBDIR)"
-	$(INSTALL_DATA) \
-	  VERSION \
-	  "$(INSTALL_LIBDIR)"
 	$(MAKE) -C runtime install
 	$(INSTALL_PROG) ocaml "$(INSTALL_BINDIR)/ocaml$(EXE)"
 ifeq "$(INSTALL_BYTECODE_PROGRAMS)" "true"
